api/data_explorer/controllers/export_url_controller.py

```diff
@@ -1,13 +1,29 @@
 import connexion
-import six
 import json
+import os
+import six
 
 from flask import current_app
+from werkzeug.exceptions import BadRequest
 
 from data_explorer.models.export_url_response import ExportUrlResponse  # noqa: E501
 
 
 def export_url_post():  # noqa: E501
+    config_path = os.path.join(current_app.config['DATASET_CONFIG_DIR'],
+                               'deploy.json')
+    if not os.path.isfile(config_path):
```

Comment:
Not sure I understand why this is being checked again. It's already done on startup and if `deploy.json` doesn't exist `current_app.config['EXPORT_URL_GCS_BUCKET']` is set to `''`, so shouldn't the check below suffice?

Reply:
This will give a more specific error message. If `deploy.json` doesn't exist, the error "deploy.json not found" is a little more helpful than "Bucket not found".

Reply:
Gotcha yeah that makes sense.